lib/common/exceptions/internal/evss_forbidden.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Common
```

Comment:
Maybe "UnexpectedForbiddenException" to line up with proposed "UnexpectedRecordNotFound"?